From 53172b4429404129b4ba72156ac6ee1ed9bf5452 Mon Sep 17 00:00:00 2001
From: Chia-Wei Wang <chiawei_wang@aspeedtech.com>
Date: Tue, 29 Aug 2023 14:14:23 +0800
Subject: [PATCH] arm: aspeed: Update memory layout

Update the memory layout based on the 1GB DRAM
of Aspeed AST2600/AST2700 EVBs.

Signed-off-by: Chia-Wei Wang <chiawei_wang@aspeedtech.com>
---
 core/arch/arm/plat-aspeed/conf.mk | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/core/arch/arm/plat-aspeed/conf.mk b/core/arch/arm/plat-aspeed/conf.mk
index ed7856bc3..a45ccca95 100644
--- a/core/arch/arm/plat-aspeed/conf.mk
+++ b/core/arch/arm/plat-aspeed/conf.mk
@@ -14,8 +14,8 @@ CFG_NUM_THREADS ?= $(CFG_TEE_CORE_NB_CORE)
 CFG_DRAM_BASE ?= 0x80000000
 CFG_DRAM_SIZE ?= 0x40000000
 
-CFG_TZDRAM_START ?= 0x88000000
-CFG_TZDRAM_SIZE ?= 0x01000000
+CFG_TZDRAM_START ?= 0xb0000000
+CFG_TZDRAM_SIZE ?= 0x1000000
 
 CFG_CORE_RESERVED_SHM ?= n
 else ifeq ($(PLATFORM_FLAVOR),ast2700)
@@ -34,10 +34,10 @@ $(call force,CFG_LPAE_ADDR_SPACE_BITS,36)
 CFG_NUM_THREADS ?= $(CFG_TEE_CORE_NB_CORE)
 
 CFG_DRAM_BASE ?= 0x400000000
-CFG_DRAM_SIZE ?= 0x400000000
+CFG_DRAM_SIZE ?= 0x40000000
 
-CFG_TZDRAM_START ?= 0x400400000
-CFG_TZDRAM_SIZE ?= 0x400000
+CFG_TZDRAM_START ?= 0x430080000
+CFG_TZDRAM_SIZE ?= 0x1000000
 
 CFG_CORE_RESERVED_SHM ?= n
 
-- 
2.25.1

